src/main/java/org/opendatakit/suitcase/net/SyncWrapper.java

```diff
@@ -60,8 +60,6 @@ public void init(CloudEndpointInfo cloudEndpointInfo) throws IOException, JSONEx
       String host = url.getHost();
 
       this.sc.init(host, this.cloudEndpointInfo.getUserName(), this.cloudEndpointInfo.getPassword());
-      
-      updateTableList();
```

Comment:
Why was this removed?

Reply:
I have removed it from here to login task. After syncWrapper is initialized we can call update table list from login task and login task will also update the status is the progress bar. This is done because after login is done from saved credentials table id list will have to be updated and user needs to know that table list is being updated as initially the table list will be blank. If saved credentials are incorrect the method would through exception, we can handle the error in login task and send user back to login page.